ansible.cfg

```diff
@@ -3,6 +3,9 @@ library = plugins/modules
 module_utils = plugins/module_utils
 doc_fragment_plugins = plugins/doc_fragments
 filter_plugins = plugins/filter
+inventory_plugins = plugins/inventory
+callback_plugins = plugins/callback
```

Comment:
You should not need that if you use it from the installed collection.

Reply:
honestly i have been thinking of having one ansible.cfg for use with and one without the collection, respectively.
Mixing them is quite fuzzy about where the modules/plugins are looked up.

Reply:
agree, but I need that if I want to test outside the collection -- which I think I want for now because otherwise I can't test on 2.7 (and 2.8)

Reply:
Do we test on 2.7/2.8 now?

Reply:
no, 2.9+

Reply:
the inventory on 2.9+, the other on 2.7+
and because of that the test suite itself runs outside the collection.